app/models/library_entry.rb

```diff
@@ -228,4 +228,15 @@ def myanimelist_linked_account
       type: 'LinkedAccount::MyAnimeList'
     )
   end
+
+  after_commit ->() { create_stat }, on: :create
```

Comment:
That's... actually not a bad name 🤔 That said, it might be better to just toss them in a block like this, depending on how much logic you'll have in there (and how many of these you need):

```ruby
after_commit(on: :create) do
  Stat::AnimeGenreBreakdown.increment_genres(user, media.genres)
end
```